### PR TITLE
New: SVG Symbol component

### PR DESCRIPTION
**Pull request by vinteo**
image

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -4,7 +4,8 @@
   "main": "dist/alexandria-main.js",
   "dependencies": {
     "lodash": "^3.0.0",
-    "react": "^0.14.0"
+    "react": "^0.14.0",
```
Comment (omgaz) [inline bower.json]: This can now be 0.14.7 as well, like everywhere else?

File: src/styles/alexandria/SvgSymbol.scss [hunk 2]
```diff
@@ -0,0 +1,7 @@
+@import '../variable';
+
+.svg-symbol-component {
+  display: inline-block;
```
Comment (omgaz) [inline src/styles/alexandria/SvgSymbol.scss]: Do we want this component to always default to inline-block?

File: src/styles/variable.scss [hunk 3]
```diff
@@ -1,3 +1,4 @@
 $spacing-etalon: 30px;
+$svg-icon-size: 70px;
```
Comment (camjc) [inline src/styles/variable.scss]: our normal icons are 16, so it probably doesn't make sense for this to be the default?

File: src/styles/alexandria/SvgSymbol.scss (hunk 2 above)
Comment (camjc) [inline src/styles/alexandria/SvgSymbol.scss]: probably.

**Comment by camjc** [comment]
Looking damn cool. Thanks for doing this mate! Heck yeah. :trophy: 
Now we've gotta go in and replace all uses with this haha.

**Comment by vinteo** [comment]
@camjc @omgaz merge?

**Comment by omgaz** [comment]
LGTMerging :ramen:

**Pull request merged**